certbot/tests/cli_test.py

```diff
@@ -63,7 +65,8 @@ def _help_output(self, args):
         return output.getvalue()
 
     @mock.patch("certbot.cli.flag_default")
-    def test_cli_ini_domains(self, mock_flag_default):
+    @test_util.patch_get_utility()
```

Comment:
Rather than decorating and adding a `unused_mock_get_utility` parameter to every method, why don't we just modify the `parse` and `_help_output` methods. The way I'd do this is the following:

1. Rename the current `parse` method to `_unmocked_parse`.
2. Add a new static `parse` method that calls `_unmocked_parse` after mocking `getUtility` using `patch_get_utility`.
3. Modify `_help_output` so `notification` messages are written to the mock stdout created in `_help_output`. You could do this by modifying the `side_effect` attribute of the `notification` method of the mock object.